packages/xdl/src/project/Doctor.ts

```diff
@@ -435,24 +435,12 @@ async function validateAsync(
     return NO_ISSUES;
   }
 
-  let exp, pkg;
-  try {
-    const config = getConfig(projectRoot, {
-      strict: true,
-      skipSDKVersionRequirement,
-    });
-    exp = config.exp;
-    pkg = config.pkg;
-    ProjectUtils.clearNotification(projectRoot, 'doctor-config-json-not-read');
-  } catch (e) {
-    ProjectUtils.logError(
-      projectRoot,
-      'expo',
-      `Error: could not load config json at ${projectRoot}: ${e.toString()}`,
-      'doctor-config-json-not-read'
-    );
-    return ERROR;
-  }
+  const { exp, pkg } = getConfig(projectRoot, {
+    strict: true,
+    skipSDKVersionRequirement,
+  });
+
+  ProjectUtils.clearNotification(projectRoot, 'doctor-config-json-not-read');
```

Comment:
So this just throws now if config can't be read? Seems better than wrapping the original error with "could not load config json..."

But I wonder if throwing will cause an `UnhandledPromiseRejectionWarning` here:
https://github.com/expo/expo-cli/blob/2c51e2f7b549ef6e2b3cb86f721959ad01ae6c5d/packages/xdl/src/project/ManifestHandler.ts#L120

Reply:
Good call, I'm now catching the error in the middleware and logging it like before